packages/near-api-js/src/account.ts

```diff
@@ -253,7 +253,7 @@ export class Account {
         }, []);
         this.printLogsAndFailures(signedTx.transaction.receiverId, flatLogs);
 
-        if (!returnError && typeof result.status === 'object' && typeof result.status.Failure === 'object') {
+        if (!returnError && typeof result.status === 'object' && typeof result.status.Failure === 'object'  && result.status.Failure !== null) {
```

Comment:
The number of `result.status.Failure` that pass this check go down after this change. Do you mean to use `||`?

Reply:
@esaminu 
No here we have to use `&&` because `typeof null` is 'object'. `result.status.Failure = null` means there is no Failure so it should not go to error block.

Reply:
Thanks for clarifying! Can we add something like this?

```suggestion
       // Should be falsy if result.status.Failure is null
        if (!returnError && typeof result.status === 'object' && typeof result.status.Failure === 'object'  && result.status.Failure !== null) {
```
Or it would be even better if we can mock and test this but not necessary if we don't already have tests that mock this imo.